sys/ztimer/auto_init.c

```diff
@@ -166,4 +196,22 @@ void ztimer_init(void)
     ZTIMER_MSEC->required_pm_mode = CONFIG_ZTIMER_MSEC_REQUIRED_PM_MODE;
 #  endif
 #endif
+
+#if MODULE_ZTIMER_SEC
+#  if MODULE_ZTIMER_PERIPH_RTC
+    LOG_DEBUG("ztimer_init(): initializing rtc\n");
+    ztimer_periph_rtc_init(&_ztimer_periph_timer_rtc_sec);
+#  else
+    LOG_DEBUG("ztimer_init(): ZTIMER_SEC convert_frac from %lu to 1\n",
+              (long unsigned)ZTIMER_SEC_CONVERT_LOWER_FREQ);
+    ztimer_convert_frac_init(&_ztimer_convert_frac_sec,
+                             ZTIMER_SEC_CONVERT_LOWER,
+                             FREQ_1HZ, ZTIMER_SEC_CONVERT_LOWER_FREQ);
+#  endif
+#  ifdef MODULE_PM_LAYERED
+    LOG_DEBUG("ztimer_init(): ZTIMER_SEC setting required_pm_mode to %i\n",
+              CONFIG_ZTIMER_SEC_REQUIRED_PM_MODE);
+    ZTIMER_SEC->required_pm_mode = CONFIG_ZTIMER_SEC_REQUIRED_PM_MODE;
```

Comment:
@kfessel: Can you take a look at https://github.com/RIOT-OS/RIOT/pull/15911 I think this needs to be applied to the hardware clock. In case of the RTC, this is already the hardware clock, but in case of RTT, it should be applied to the RTT clock sitting below the convert frac clock.

Reply:
you are right and while writing these lines i thought this is wrong but its the current way to go and i was already searching if there is an issue or PR in this direction.
so these PRs are in "conflict". one has to merged the other has to follow